xee/ext.py

```diff
@@ -231,19 +231,31 @@ def __init__(
     # Parse the dataset bounds from the native projection (either from the CRS
     # or the image geometry) and translate it to the representation that will be
     # used for all internal `computePixels()` calls.
-    try:
-      if isinstance(geometry, ee.Geometry):
+    if geometry is None:
+      try:
+        x_min_0, y_min_0, x_max_0, y_max_0 = self.crs.area_of_use.bounds
+      except AttributeError:
+        # `area_of_use` is probable `None`. Parse the geometry from the first
+        # image instead (calculated in self.get_info())
         x_min_0, y_min_0, x_max_0, y_max_0 = _ee_bounds_to_bounds(
             self.get_info['bounds']
         )
-      else:
-        x_min_0, y_min_0, x_max_0, y_max_0 = self.crs.area_of_use.bounds
-    except AttributeError:
-      # `area_of_use` is probable `None`. Parse the geometry from the first
-      # image instead (calculated in self.get_info())
+    elif isinstance(geometry, ee.Geometry):
       x_min_0, y_min_0, x_max_0, y_max_0 = _ee_bounds_to_bounds(
           self.get_info['bounds']
       )
+    elif isinstance(geometry, Union[List, Tuple, np.ndarray]):
```

Comment:
Does typing.Sequence work here?

Consider adding a check for length of the sequence, too.

Reply:
Ah! You have one :)